deploy/crds/kieapp.crd.yaml

```
@@ -1913,6 +1913,31 @@ spec:
                                 type: string
                               type: array
                           type: object
+                        kafkaJbpm:
```

To make configuration cleaner, I would suggest to use the whole name. kafkaJbpm mayn not be clear and in some cases confusing, e.g. users would set up kafkaJbpm and expects that the messages from process would be handled.
```suggestion
                        kafkaJbpmEventEmitters:
```

---

ok